assets/src/js/_acf.js

```diff
@@ -855,6 +855,148 @@
 		return obj;
 	};
 
+	/**
+	 * Check if an object has only numeric string keys.
+	 * Used to detect objects that should be converted to arrays (e.g., checkbox values).
+	 *
+	 * @since   SCF 6.6.0
+	 *
+	 * @param   object obj The object to check
+	 * @return  boolean True if all keys are numeric strings
+	 */
+	acf.hasOnlyNumericKeys = function ( obj ) {
```

Comment:
How should this handle edge cases like `00123` or negative values? Maybe that can't happen in the functionality currently using this function, but for such a global function, it would be good to clarify it.

It's probably something `acf` itself shouldn't handle as a public API.

Reply:
Thanks for making it private 👍